builders/integrations/oracles/dia.md

```diff
@@ -0,0 +1,139 @@
+
+# 🪙 Token Price Feeds
+
+DIA token price feeds provide smart contracts with real-time price information for [3,000+ cryptocurrencies](https://diadata.org/app/price), sourced transparently from [90+ trusted, high-volume DEXs and CEXs](https://diadata.org/app/source/defi).
+
+## How to access DIA oracles?
+
+Here is an example of how to access a price value on DIA oracles:
+
+1. Access your custom oracle smart contract on Moonbeam.
+2. Call `getValue(pair_name)` with `pair_name` being the full pair name such as `BTC/USD`. You can use the "Read" section on the explorer to execute this call.
+3. The response of the call contains two values:
+- The current asset price in USD with a fix-comma notation of 8 decimals.
+- The UNIX timestamp of the last oracle update.
+
+You can find DIA's oracle integration samples in Solidity and Vyper languages by visiting:
+
+→ [Access the Oracle | DIA Documentation](https://docs.diadata.org/products/token-price-feeds/access-the-oracle)
+
+## Moonbeam demo price oracles
+
+DIA has deployed the following demo oracles for the Moonbeam community. It provides a limited selection of cryptocurrency price feeds with predefined configuration settings.
+
+> ⚠️ NOTE: DIA demo oracles are not intended for use in production environments. Developers can request a dedicated, production-ready oracle with custom price feeds and configuration settings. Start the request process: [Request a Custom Oracle | DIA Documentation](https://docs.diadata.org/introduction/intro-to-dia-oracles/request-an-oracle)
+
+### Demo Oracle Smart Contracts
+
+| Network        | Contract address      
+| -------------- | -------------------------------------------------------------------------------------------------------------------------------------- |
+| Moonbeam Alpha  | [`0xe23d8713aa3a0a2c102af772d2467064821b8d46`](https://moonbase.moonscan.io/address/0xe23d8713aa3a0a2c102af772d2467064821b8d46)        |
+| Moonriver  | [`0x11f74b94afb5968119c98ea277a2b73208bb39ab`](https://moonriver.moonscan.io/address/0x11f74b94afb5968119c98ea277a2b73208bb39ab)        |
+
+### Included Price Feeds
+
+[DIA/USD](https://diadata.org/app/price/asset/Ethereum/0x84cA8bc7997272c7CfB4D0Cd3D55cd942B3c9419/), [BTC/USD](https://diadata.org/app/price/asset/Bitcoin/0x0000000000000000000000000000000000000000/), [USDC/USD](https://diadata.org/app/price/asset/Ethereum/0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48/)
+
+## Supported token API endpoints
+
+DIA also supports API and GraphQL endpoints to return cryptocurrency price data. You can [visit the DIA Documentation](https://docs.diadata.org/products/token-price-feeds/access-api-endpoints) to see all API endpoints.
+
+# 🎨 NFT Floor Price Feeds
+
+DIA NFT floor price feeds provide smart contracts with real-time price information of [18,000+ NFT collections](https://diadata.org/app/floor-price), sourced on-chain with 100% transparency from [multiple, cross-chain NFT marketplaces](https://diadata.org/app/source/nft).
```

Comment:
Do you currently have any NFT collections from Moonbeam or Moonriver? I searched a few and nothing came up, but yeah just curious 🙂